support-frontend/assets/pages/weekly-subscription-landing/components/hero/hero.tsx

```diff
@@ -19,73 +19,90 @@ import GridImage from 'components/gridImage/gridImage';
 import Hero from 'components/page/hero';
 import OfferStrapline from 'components/page/offerStrapline';
 import PageTitle from 'components/page/pageTitle';
-import type { Participations } from 'helpers/abTests/abtest';
+import type { IsoCountry } from 'helpers/internationalisation/country';
 import type { CountryGroupId } from 'helpers/internationalisation/countryGroup';
 import {
 	detect,
 	GBPCountries,
 } from 'helpers/internationalisation/countryGroup';
+import type { ProductPrices } from 'helpers/productPrice/productPrices';
 import { promotionHTML } from 'helpers/productPrice/promotions';
 import type { PromotionCopy } from 'helpers/productPrice/promotions';
 import { sendTrackingEventsOnClick } from 'helpers/productPrice/subscriptions';
 import { guardianWeeklyHeroBlue } from 'stylesheets/emotion/colours';
+import WeeklyProductPrices from '../weeklyProductPrices';
 
-type PropTypes = {
+type WeeklyHeroPropTypes = {
 	orderIsAGift: boolean;
-	countryGroupId: CountryGroupId;
 	promotionCopy: PromotionCopy;
-	participations: Participations;
 };
-const weeklyHeroCopy = css`
-	padding: 0 ${space[3]}px ${space[3]}px;
-	color: ${text.primary};
-`;
-const weeklyHeroTitle = css`
-	${headline.small({
-		fontWeight: 'bold',
-	})};
-	margin-bottom: ${space[3]}px;
-
-	${from.mobileLandscape} {
-		width: 75%;
-	}
 
-	${from.tablet} {
-		${headline.large({
+type PriceCardsWeeklyHeroPropTypes = {
+	orderIsAGift: boolean;
+	promotionCopy: PromotionCopy;
+	countryId: IsoCountry;
+	productPrices: ProductPrices;
+	isPriceCardsAbTestVariant: boolean;
+};
+
+const styles = {
+	weeklyHeroCopy: css`
+		padding: 0 ${space[3]}px ${space[3]}px;
+		color: ${palette.neutral[7]};
+	`,
+	weeklyHeroTitle: css`
+		${headline.small({
 			fontWeight: 'bold',
 		})};
+		margin-bottom: ${space[3]}px;
+
+		${from.mobileLandscape} {
+			width: 75%;
+		}
+
+		${from.tablet} {
+			${headline.large({
+				fontWeight: 'bold',
+			})};
+			width: 100%;
+		}
+	`,
+	pageTitleOverrides: css`
 		width: 100%;
-	}
-`;
-const weeklyHeroParagraph = css`
-	${body.medium({
-		lineHeight: 'loose',
-	})}
-	margin-bottom: ${space[9]}px;
-
-	/* apply the same margin to paragraphs parsed from markdown from promo codes */
-	& p:not(:last-of-type) {
+	`,
+	weeklyHeroParagraph: css`
+		${body.medium({
+			lineHeight: 'loose',
+		})}
 		margin-bottom: ${space[9]}px;
-	}
-`;
-const showOnMobile = css`
-	display: block;
 
-	${from.mobileLandscape} {
-		display: none;
-	}
-`;
+		/* apply the same margin to paragraphs parsed from markdown from promo codes */
+		& p:not(:last-of-type) {
```

Comment:
Does this change the bottom margin in the h2 title parent header? Its not a paragraph, presume last-of-type affects title?